### PR TITLE
RFC 136: Remove the CSS build system

### PR DESCRIPTION
**Pull request by foolip**
[Rendered](https://github.com/web-platform-tests/rfcs/blob/css_build_system/rfcs/css_build_system.md)

### CODE REVIEW

**Review by jgraham — APPROVED** [review]
This needs to be fleshed out more, clearly, with specific details about what will be removed, but in general I"m strongly in favour of this and don't expect to object to any of those details.

File: rfcs/css_build_system.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Summary
+
+Remove css/tools/ and the special lint rules for css/.
+
+## Details
+
+http://test.csswg.org/suites/ hasn't been updated since January.
+
+TODO: does the CSSWG want to keep the spec link rule?
```
Comment (jgraham) [inline rfcs/css_build_system.md:11]: I would proceed for now on the assumption the answer is "yes", but it's worth asking the question. If not we can always remove the requirement in a second step.

File: rfcs/css_build_system.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# RFC 136: Remove the CSS build system
+
+## Summary
+
+Remove css/tools/ and the special lint rules for css/.
+
+## Details
+
+http://test.csswg.org/suites/ hasn't been updated since January.
```
Comment (gsnedders) [inline rfcs/css_build_system.md:9]: https://github.com/w3c/csswg-drafts/issues/6896 makes it sound like there have been serious problems there for much longer. (And indeed I get the error given in https://github.com/w3c/csswg-drafts/issues/6896#issuecomment-996047519 when I try to build it locally.)

File: rfcs/css_build_system.md (hunk 1 above)
Comment (foolip) [reply rfcs/css_build_system.md:11]: I've done it in a separate commit. Let's just say it's not part of this RFC to reduce risk.

**Comment by gsnedders** [comment]
CSS WG [resolved](https://github.com/w3c/csswg-drafts/issues/6896#issuecomment-1499457107), so we can proceed with this.

**Review by Ms2ger — APPROVED** [review]

**Comment by jgraham** [comment]
@foolip can you update this to include the detailed plan, and remove the TODOs?

**Comment by foolip** [comment]
@jgraham it is done!

**Review by jgraham — APPROVED** [review]

**Comment by foolip** [comment]
I've marked https://github.com/web-platform-tests/wpt/pull/38976 as ready for review now.

**Pull request merged**